js/README.md

```diff
@@ -0,0 +1,73 @@
+# Solido JS SDK
+
+Typescript SDK to facilitate interacting with the Solido program in JS/TS applications
+
+## Installation
+
+### Yarn
+
+```
+$ yarn add @chorusone/solido.js
+```
+
+### npm
+
+```
+$ npm install @chorusone/solido.js
+```
+
+## Development
+
+### Install packages
+```
+$ npm install
+or
+$ yarn
+```
+
+### Run TS watch script
+```
+$ npm run watch:ts
+```
+
+### Run tests watch script
+```
+$ npm run watch:test
+```
+
+### Build and Release
+TODO
+
+
+## Usage
+
+### Javascript
+```js
+const SolidoJS = require("@chorusone/solido.js");
+const snapshot = SolidoJS.getSnapshot(...params);
```

Comment:
We should give a full example that works. I think it needs a connection and also the `SolidoJS.MAINNET_PROGRAM_ADDRESSES`, right?